boilerplate/babel.config.js

```diff
@@ -25,31 +25,12 @@ const plugins = [
   "react-native-reanimated/plugin", // NOTE: this must be last in the plugins
 ]
 
-const vanillaConfig = {
-  presets: ["module:metro-react-native-babel-preset"],
-  env: {
-    production: {},
-  },
-  plugins,
-}
-
-const expoConfig = {
+const babelConfig = {
   presets: ["babel-preset-expo"],
   env: {
     production: {},
   },
   plugins,
 }
 
-let isExpo = false
-try {
-  const Constants = require("expo-constants")
-  // True if the app is running in an `expo build` app or if it's running in Expo Go.
-  isExpo =
-    Constants.executionEnvironment === "standalone" ||
-    Constants.executionEnvironment === "storeClient"
-} catch {}
-
-const babelConfig = isExpo ? expoConfig : vanillaConfig
```

Comment:
Why are these babel configs getting pulled out?

Reply:
I think that's why these tests are failing: https://app.circleci.com/pipelines/github/infinitered/ignite/2492/workflows/f1b88990-c262-4145-bd1e-4bd270df4d31/jobs/3075